bin/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   parserOptions: {
-    ecmaVersion: 6,
```

Comment:
I wonder what is the reason for having a separate `.eslintrc` here. I guess there would be too many linting errors from the past, based on the default `.eslintrc`.

We should aim for replacing this eslintrc with the default one. Not in this PR though.

Reply:
Resolved in https://github.com/Trustroots/trustroots/pull/1042